packages/poaps/docs/Inputs.md

```diff
@@ -0,0 +1,84 @@
+# Input Types Documentation
+
+This section documents the input types defined for fetching and managing POAPs .
+
+## PoapsSortFields
+
+The `PoapsSortFields` enumeration defines the available fields by which POAPs can be sorted.
+
+| Property   | Value       | Description                                            |
+| ---------- | ----------- | ------------------------------------------------------ |
+| `MintedOn` | `minted_on` | Represents sorting by the date when a POAP was minted. |
+| `Id`       | `id`        | Represents sorting by the ID of a POAP.                |
+
+```typescript
+export enum PoapsSortFields {
+  MintedOn = 'minted_on',
+  Id = 'id',
+}
+```
+
+## FetchPoapsInput
+
+The `FetchPoapsInput` interface represents the input fields for fetching POAPs and extends `PaginationInput` to provide pagination capability.
+
+| Property              | Type               | Description                                                |
+| --------------------- | ------------------ | ---------------------------------------------------------- |
+| `name`                | `string?`          | Optional filter for the name of a POAP.                    |
+| `chain`               | `Chain?`           | Optional filter for the blockchain chain of a POAP.        |
+| `mintedDateFrom`      | `string?`          | Optional filter for the start date when a POAP was minted. |
+| `mintedDateTo`        | `string?`          | Optional filter for the end date when a POAP was minted.   |
+| `ids`                 | `number[]?`        | Optional filter for specific POAP IDs.                     |
+| `collectorAddress`    | `string?`          | Optional filter for the collector's address.               |
+| `dropId`              | `number?`          | Optional filter for a specific drop ID.                    |
+| `sortField`           | `PoapsSortFields?` | Field by which to sort the results.                        |
+| `sortDir`             | `Order?`           | Direction in which to sort the results.                    |
```

Comment:
Can a link to the Order object could be added here and elsewhere?

Reply:
So now there is no way of knowing in the documentation what to pass there?

Reply:
Not yet